src/data.js

```diff
@@ -1312,6 +1312,27 @@ module.exports = [
       'Photographer',
     ],
   },
+  {
+    name: 'Ed Henderson',
+    description: 'Unicorn amongst donkeys',
+    url: 'https://twitter.com/edhenderson',
```

Comment:
the URL has to be a ` /uses`

Reply:
I actually hadn't even put a URL in originally but it insisted on one.